### PR TITLE
change mozillavpn_cirrus to mozillavpn_backend_cirrus

### PR DESCRIPTION
**Pull request by Marlene-M-Hirose**
change app_name, app_id, and v1_name in repositories.yaml

### CODE REVIEW

**Review by sean-rose — APPROVED** [review]

File: repositories.yaml [hunk 1]
```diff
@@ -983,8 +983,8 @@ applications:
       expiration_policy:
         delete_after_days: 180
     channels:
-      - v1_name: mozillavpn-cirrus
-        app_id: mozillavpn_cirrus
+      - v1_name: mozillavpn-backend-cirrus
+        app_id: mozillavpn_backend_cirrus
```
Comment (yashikakhurana) [inline repositories.yaml:987]: 👍

File: repositories.yaml [hunk 2]
```diff
@@ -965,7 +965,7 @@ applications:
         additional_dependencies:
           - glean-core
 
-  - app_name: mozillavpn_cirrus
+  - app_name: mozillavpn_backend_cirrus
```
Comment (yashikakhurana) [inline repositories.yaml:968]: 👍

**Review by yashikakhurana — APPROVED** [review]

**Comment by badboy** [comment]
This will create new tables, won't it? But they are empty, so we can just drop them and I assume data, if any, has already been sent with the updated app ID anyway?
(Only other thing then is that we're now running into the change freeze today)

**Comment by sean-rose** [comment]
> This will create new tables, won't it? But they are empty, so we can just drop them and I assume data, if any, has already been sent with the updated app ID anyway? (Only other thing then is that we're now running into the change freeze today)

Yes, the live & stable tables for the `mozillavpn_cirrus` app ID are still empty.

The [DE change freeze](https://docs.google.com/document/d/1bK9-nx_mAAb_-vhZ0NImmLZ2Nk1gM7nhsqHJ87kguDI/edit) does specifically call out "no new Glean apps", which this app rename would technically qualify as, so I guess this will have to wait until Jan 3 (sorry @brizental).

**Comment by yashikakhurana** [comment]
@sean-rose just wondering can we merge this now?

**Comment by sean-rose** [comment]
> @sean-rose just wondering can we merge this now?

In terms of the change freeze we're clear to merge, but we still need a code owner to approve this PR.  I've [asked in Slack](https://mozilla.slack.com/archives/C01E8GDG80N/p1704403354838329).

**Review by travis79 — APPROVED** [review]

**Pull request merged**